samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -192,9 +193,9 @@ Test not discoverable advertising by completing `Testing`_ and the following add
    a. Go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`) > :guilabel:`Devices` > :guilabel:`Saved devices`.
    #. Verify that the paired device is appearing on the list.
 
-#. Press **Button 1** to switch to the Fast Pair not discoverable advertising show UI indication mode.
-   **LED 3** starts blinking rapidly.
-   If you want to test the Fast Pair not discoverable advertising hide UI indication mode, press **Button 1** again.
+#. Make sure that the Fast Pair Provider is in the Fast Pair not discoverable advertising show UI indication mode.
+   **LED 3** must blink rapidly.
+   If you want to test the Fast Pair not discoverable advertising hide UI indication mode, press **Button 1**.
```

Comment:
This is not clear. How one can make sure that the FPP is in the right mode? What if LED 3 is not blinking rapidly? Is this the result of the previous step, that is adding the FPP to the Android device list?

Reply:
This is a good point @greg-fer .

Also, My initial thought was that after Steps 1 and 2, the FPP is in the not discoverable show UI indication advertising mode. But I think that is not the case. 
Isn't it true that The FFP is in this mode only after FPP is added on the second device (after step 4) because we have requirement that - _Testing not discoverable advertising requires using at least two Android devices registered to the same Google account. The first one needs to be bonded with Fast Pair Provider before the not discoverable advertising can be detected by the second one._ : http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8330/nrf/samples/bluetooth/peripheral_fast_pair/README.html#not-discoverable-advertising-requirements .

Reply:
Yes, as a consequence of previous step, advertising mode should switch to `not discoverable advertising show UI indication mode` which is denoted by LED 3 blinking rapidly. User should make sure that it really happened (it will happen only if Provider successfully pairs in previous step).

Reply:
If LED 3 didn't blink rapidly, advertising mode can be toggled by Button 1, but that won't be the case if previous step is successful.